gen_errata.rb

```diff
@@ -455,6 +456,20 @@ def add_binary_packages(errata, packages, releases: ['stretch'], architecture_wh
               packages[p[:name]],
               architecture_whitelist
             )
+            #  On Debian, there are differnet source packages which are used to build the linux kernel packages
+            #    linux -> is used to create the unsigned linux images
+            #    linux-signed-XX (e.g. linux-signed-amd64) is used to build signed linux image which runs on secureboot enabled systems
+            if erratum.source_package == 'linux' && special_kernel_pkg_collection&.include?(p[:release])
+              architecture_whitelist.each do |arch|
+                source_pkg = "linux-signed-#{arch}"
+                new << get_binary_packages_for_erratum_package(
```

Comment:
```suggestion
                new += get_binary_packages_for_erratum_package(
```
the method returns an Array, so we need to concatenate it to the existing one.